docs/methods/machine_learning/nnp.md

```diff
@@ -1,6 +1,20 @@
 # Neural Network Potentials
 
-Unfortunately no one has gotten around to writing this page yet :-(
+## Input Section
+
+NNPs in CP2K are supported via the `NNP` Force_Eval section. As an example, a basic `NNP` section would be:
```

Comment:
One can link directly to input sections like this: `[NNP](#CP2K_INPUT.FORCE_EVAL.NNP)` and to individual keywords like this: `[NNP_INPUT_FILE_NAME](#CP2K_INPUT.FORCE_EVAL.NNP.NNP_INPUT_FILE_NAME)`.

Reply:
perfect, thanks! I'll prepare a more substantial documentation in the next days.